vlingo-reactive-messaging-patterns/src/main/java/io/vlingo/reactive/messaging/patterns/messagefilter/OrderPlaced.java

```diff
@@ -0,0 +1,25 @@
+// Copyright © 2012-2018 Vaughn Vernon. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the
+// Mozilla Public License, v. 2.0. If a copy of the MPL
+// was not distributed with this file, You can obtain
+// one at https://mozilla.org/MPL/2.0/.
+
+package io.vlingo.reactive.messaging.patterns.messagefilter;
+
+public class OrderPlaced {
```

Comment:
Since you already have an `InventorySystem` with an `orderPlaced(...)` protocol method, I suggest refactoring to `orderPlaced(final Order order)` and delete this class. I think this class was only necessary to deal with Akka's typeless messaging.